lib/fontcustom/scripts/generate.py

```diff
@@ -126,16 +127,12 @@ def createGlyph( name, source, code ):
     manifest['fonts'].append(fontfile + '.woff')
 
     # Convert EOT for IE7
-    subprocess.call('python ' + scriptPath + '/eotlitetool.py ' + fontfile + '.ttf -o ' + fontfile + '.eot', shell=True)
-    # check if windows
-    if os.name == 'nt':
-        subprocess.call('move ' + fontfile + '.eotlite ' + fontfile + '.eot', shell=True)
-    else:
-        subprocess.call('mv ' + fontfile + '.eotlite ' + fontfile + '.eot', shell=True)
+    subprocess.call(['python', scriptPath + '/eotlitetool.py', fontfile + '.ttf', '-o', fontfile + '.eot'])
+    shutil.move(fontfile + '.eotlite', fontfile + '.eot')
     manifest['fonts'].append(fontfile + '.eot')
 
     # Convert TTF to WOFF2
-    subprocess.call('woff2_compress \'' + fontfile + '.ttf\'', shell=True)
+    subprocess.call(['woff2_compress', fontfile + '.ttf'])
```

Comment:
calling without shell and remove `'` makes woff2 work on Windows. apostrophes are not supported by cmd.exe